src/azure-cli/azure/cli/command_modules/sql/_validators.py

```diff
@@ -82,21 +82,31 @@ def validate_subnet(cmd, namespace):
     from msrestazure.tools import resource_id, is_valid_resource_id
     from azure.cli.core.commands.client_factory import get_subscription_id
 
-    subnet = namespace.virtual_network_subnet_id
+    is_instance_pool = False
```

Comment:
Consider adding comment explaining this is just because of the different custom function arg names